contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,142 @@
+
+We felt that the actual problem we were solving was still prioritization of where to focus time.
+
+We were both big believers in [Andrew Grove](https://en.wikipedia.org/wiki/Andrew_Grove)'s [High Output Management](https://www.amazon.com/High-Output-Management-Andrew-Grove/dp/0679762884/ref=sr_1_1?dchild=1&keywords=high+output+management&qid=1610712757&s=books&sr=1-1).
+
+By this stage, we were thinking we just wanted to work with people that would at least try our stuff.
+
+### 4. technical debt monitoring tool using surveys after each pull request
+
+We finally started getting a little clearer over those we were build
+
+It turns out everyone has problems with technical debt, but solving it involves changing how teams prioritize.
+
+We just weren't the right people to run this business. Although Tim had struggled with technical debt first hand, neither of us had struggled with actually solving it. If one of us had managed an engineering team before, we'd have perhaps been better placed to understand this area.
+
+Along the way though, we learned a lot about how developers and product managers work together. We'd also created a large list of
+
+If you can't stop thinking of other ideas, you probably are building something you don't like. 
+
+### 5. engineering retention tool using surveys after each pull request
+
+Those capricious engineers coming and going like ships in the night.
+
+This idea didn't really come from us, which probably doomed it before it even really started. Amusingly we had to do a demo day dry run for this in front of 500 people who made up the YC batch.
+
+This lasted all of about 5 days. We had a bunch of meetings left over from (4).
+
+We had a wildly unenthusiastic response. The lowlight was during one of the meetings that we resorted to asking the CTO of an 80 person start up what his biggest problem was, "I've not really got any". Cool, we'll just run out of food and shelter then.
+
+### 6. open source product analytics platform
+
+Things were about to get meta.
+
+Along the winding journey above, we got frustrated having to send all our user data to 3rd parties. It felt wrong and it meant we'd lose a bunch of user data that would have been quite useful.
+
+## Types of problems we came up against
+
+###Local versus global maxima
+
+The hardest pivot to make was when we had the most traction. For us, that was dropping our technical debt tool.
+
+It is *crucial* to be really quick at moving on when it's time. If you're talking about pivoting, you must immediately pivot or dramatically change your approach. We realized the actual reason we had to pivot was we were the wrong people to build the right product. It was obvious our existing product wasn't working because our users told us as such.
+
+It's easy to imagine a world where we just about scraped by with the tool, raised a small seed round from oversubscribed investors, and we'd still be struggling with someothing we weren't enjoying working on.
```

Comment:
"something"

I guess the bot didn't exactly make me obsolete just yet

Reply:
bizarre it didn't spot this